cinch/roles/jenkins_master/templates/etc/nginx/conf.d/jenkins_http.conf

```diff
@@ -1,3 +1,6 @@
+limit_req_zone $binary_remote_addr zone=jenkins:10m rate={{ nginx_request_limit }};
```

Comment:
What does this '10m' number mean?  Is this something that should be parameterized?

Reply:
This value is the size of the rate limiting state cache for this rate limit zone in nginx. 10m comes to about 160,000 ip addresses. I don't see a need to parameterize this one but am up for it if that would be valuable.

Reply:
I don't see a downside to parameterizing all the number values.  It gives users an opportunity to tweak those values if needed, and even if they are never changed by anyone, having the values documented with comments in our vars files (with links to specific nginx docs) makes it easier for cinch users to understand what's going on.